common/constants.go

```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package common
 
 const (
-	DATABASE_CR_STATUS_DELETING     = "DELETING"
```

Comment:
is it possible to apply the appropriate case to the status in the response - instead of defining new string constants?

Reply:
Database.Status.Status being a string will only show what's been assigned to it unless formatted (either by us, or by some annotation) and I'm not quite sure if that's possible through the annotations / other such kubebuilder validations.

Reply:
If it's not straightforward/trivial, I am ok to achieve it this way too. Thanks for checking.